.gitignore

```diff
@@ -30,3 +30,6 @@ bower_components
 Thumbs.db
 
 dist/css/styles.css
+*.json
+package.json
```

Comment:
We need to keep package.json in the Git repository, as this is where all the packages and project information is stored. Without it people wouldn't be able to install the required dependencies.